tests/Domain.UnitTests/Entities/Product/ValueObjects/ProductTitleValidatorTests.cs

```diff
@@ -1,56 +1,54 @@
-﻿using System.Linq;
-using CleanCompanyName.DDDMicroservice.Domain.Entities.Products;
-using CleanCompanyName.DDDMicroservice.Domain.Entities.Products.ValueObjects;
+﻿using CleanCompanyName.DDDMicroservice.Domain.Entities.Products.ValueObjects;
 using FluentValidation.TestHelper;
 using Xunit;
 
 namespace CleanCompanyName.DDDMicroservice.Domain.UnitTests.Entities.Products.ValueObjects;
 
-public class ProjectTitleValidatorTests
+public class ProductTitleValidatorTests
 {
-    private readonly ProjectTitleValidator _validator;
+    private readonly ProductTitleValidator _validator;
 
-    public ProjectTitleValidatorTests()
+    public ProductTitleValidatorTests()
     {
-        _validator = new ProjectTitleValidator();
+        _validator = new ProductTitleValidator();
     }
     
     [Fact]
     public void WHEN_Title_is_empty_THEN_give_an_error()
     {
-        var model = new ProjectTitle { Title = string.Empty };
+        var model = new ProductTitle { Value = string.Empty };
         var result = _validator.TestValidate(model);
 
         result
-            .ShouldHaveValidationErrorFor(person => person.Title);
+            .ShouldHaveValidationErrorFor(person => person.Value);
```

Comment:
Too much 🧍🧍🧍🧍🧍🧍🧍 in this test class, right? 😆 we probably need here `title`s or just `t`s.
```suggestion
            .ShouldHaveValidationErrorFor(title => title.Value);
```

Reply:
Good catch! :-D